index.html

```diff
@@ -27,6 +26,141 @@ <h1 class="section-header" id="my-name">Christian Onoh</h1>
       <p class="my-button">Start collaboration</p>
       <img class="project-img" src="assets/images/image_geometry_2.svg" alt="">
     </section>
+    <section class="section-title">
+      <header>
+        <h1 class="section-header">Projects</h1>
+      </header>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-antoni-shkraba-4348404.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 1</h2>
+        </header>
+        <ul class="project-tools">
+          <li class="project-tool">HTML/CSS</li>
+          <li class="project-tool">Ruby on Rails</li>
+          <li class="project-tool">Javascript</li>
+        </ul>
+        <p class="see-project">See this project <i class="fa fa-arrow-right fa-sm" aria-hidden="true"></i> </p>
+      </div>
+      <div class="project">
+        <img class="project-img" src="assets/images/pexels-eduardo-rosas-907487.jpg" alt="">
+        <header >
+          <h2 class="project-name">Project 2</h2>
+        </header>
```

Comment:
- [ ] Kindly as suggested in the project requirements, I think it would be better to lay out the elements in the ```Works``` section using **CSS Grid** and **Flexbox** combined (it is mandatory that you use both). Kindly, Use **CSS Grid** in the main container that contains all of the cards and use **Flexbox** to lay out an individual card. 😎😎

You may follow this[ link ](https://css-tricks.com/snippets/css/complete-guide-grid/)for more info on CSS grid 🙏